src/Psalm/Internal/Analyzer/Statements/Expression/Fetch/VariableFetchAnalyzer.php

```diff
@@ -719,6 +723,9 @@ public static function getGlobalType(string $var_id, int $codebase_analysis_php_
                 'HTTP_SEC_CH_UA_PLATFORM' => $non_empty_string_helper,
                 'HTTP_SEC_CH_UA_MOBILE'   => $non_empty_string_helper,
                 'HTTP_SEC_CH_UA'          => $non_empty_string_helper,
+                // phpunit
+                'APP_DEBUG' => $bool_string_helper,
```

Comment:
For the record
- Phpunit cast `'false'` and `'true'` to a boolean.
- Symfony support using `APP_DEBUG=0` or `APP_DEBUG=1` in the .env file.